presentation/components/src/presentation-components/properties/NavigationPropertyEditor.tsx

```diff
@@ -47,8 +49,8 @@ export const NavigationPropertyEditorContext = React.createContext<NavigationPro
 export function useNavigationPropertyEditingContextProps(imodel: IModelConnection, dataProvider: IContentDataProvider): NavigationPropertyEditorContextProps {
   return React.useMemo<NavigationPropertyEditorContextProps>(() => ({
     imodel,
-    getNavigationPropertyInfo: async (record) => {
-      const field = await dataProvider.getFieldByPropertyRecord(record);
+    getNavigationPropertyInfo: async (property) => {
+      const field = await dataProvider.getFieldByPropertyRecord(new PropertyRecord({ valueFormat: PropertyValueFormat.Primitive }, property));
```

Comment:
The `ContentDataProvider.getFieldByPropertyRecord` takes a `PropertyRecord` and uses only property description from it. Maybe we should also add a `ContentDataProvider.getFieldByPropertyDescription`? That way we wouldn't need to create a fake `PropertyRecord` here.